src/libraries/Common/src/Interop/Browser/System.Runtime.Interop.JavaScript.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think this should go to `src/libraries/System.Runtime.Interop.JavaScript/System.Runtime.Interop.JavaScript.csproj`

Reply:
addressed in recent commit